technical-guide/developer/ui.md

```diff
@@ -660,3 +660,84 @@ You can use Storybook's `<Canvas>` element to showcase specific stories to enric
 When including codeblocks, please add code in Clojure syntax (not JSX).
 
 You can find an example MDX file in the [Buttons docs](https://hourly.penpot.dev/storybook/?path=/docs/buttons-docs--docs).
+
+### Replacing and old component
+
+**Create screenshots for visual tests BEFORE making any changes.**
+
+Run the tests in the DS project. On TMUX, stop Shadow and run:
```

Comment:
- `tmux` is spelled in lowercase
- `stop Shadow`: stop the Shadow CLJS compiler in tmux tab `#1`.